src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.coffee

```diff
@@ -27,7 +27,7 @@ DashboardCompanySelectboxCtrl = ($scope, $state, $stateParams, $uibModal, MnoeCu
     selectBox.close()
 
   selectOrganization = ->
-    selectBox.organization = _.find(MnoeCurrentUser.user.organizations, { id: MnoeOrganizations.selectedId })
+    selectBox.organization = MnoeOrganizations.selected.organization || _.find(MnoeCurrentUser.user.organizations, { id: MnoeOrganizations.selectedId })
```

Comment:
Maybe use `MnoeOrganizations.getSelected()` as it was designed as a getter. `MnoeOrganizations.selected` should really be `_selected` to hint that it's designed as a private property. This means if in the future some changes a made to the way the `MnoeOrganization` service retrieves a stored selected organisation, this code can go unchanged.